app/graphql/types/media_association.rb

```diff
@@ -0,0 +1,14 @@
+class Types::MediaAssociation < Types::BaseObject
```

Comment:
Could you elaborate on why you feel MediaAssociation, MangaList, and AnimeList should exist, instead of going directly from Profile->Library->LibraryEntry?

This is kinda the GQL I was thinking would be good:

```gql
enum MediaType {
  ANIME,
  MANGA
}

type Library {
  all: LibraryEntriesConnection!,
  current: LibraryEntriesConnection!,
  planned: LibraryEntriesConnection!,
  onHold: LibraryEntriesConnection!,
  dropped: LibraryEntriesConnection!
}

type Profile {
  library(type: MediaType): Library!
}
```

Reply:
I had a discussion at some point with @matthewdias and that approach you did above was 1/2. I decided to go with having `anime` and `manga` types (sub types?) because I wasn't sure at the time how to pass parameters around, and it read nicer than having to alias all the queries.

Reply:
I would prefer `Profile->Library->LibraryEntry` too.